.github/actions/publish/action.yml

```diff
@@ -171,8 +145,8 @@ runs:
 
         for target in ${TARGETS}; do
             electron-builder ${ELECTRON_BUILDER_OS} ${target} ${ARCHITECTURE_FLAGS} \
-              --c.extraMetadata.analytics.sentry.token='${{ steps.sentry.outputs.dsn }}' \
-              --c.extraMetadata.analytics.mixpanel.token='balena-etcher' \
+              --c.extraMetadata.analytics.sentry.token='https://739bbcfc0ba4481481138d3fc831136d@o95242.ingest.sentry.io/4504451487301632' \
```

Comment:
Is it something visible in the built package or logs?
Or do we want to put it into a secret instead showing it in the action directly?
I guess there is another authentication so it cna't be spammed by anyone.

Reply:
This is not more secret than the amplitude or a google analytics token
This is trivial to extract from the build anyway and is safe to be made public